bookkeeper-server/src/test/java/org/apache/bookkeeper/util/TestUtils.java

```diff
@@ -52,4 +60,21 @@ public static boolean hasLogFiles(File ledgerDirectory, boolean partial, Integer
         }
         return result;
     }
+
+    public static void waitUtilLacUpdated(ReadHandle rh, long newLac) throws Exception {
+        long lac = rh.getLastAddConfirmed();
+        while (lac < newLac) {
+            TimeUnit.MILLISECONDS.sleep(20);
+            lac = rh.readLastAddConfirmed().get();
+        }
+    }
+
+    public static long waitUtilExplicitLacUpdated(LedgerHandle rh, long newLac) throws Exception {
```

Comment:
Why are you using ReadHandle for one and LedgerHandle for the other?

Reply:
I would prefer using new API whenever I can. readExplicitLastAddConfirmed is not available in the new API, that forces me going back to LedgerHandle.